dist/PelTec.js

```diff
@@ -35,6 +35,9 @@ export class PelTecDisplay extends DisplayWithPowerButton {
             this.configureParameter("sensor.peltec", "accessories", "optional")
             this.configureParameter("sensor.peltec", "accessories_value", "optional")
             this.configureParameter("sensor.peltec", "operation_mode", "optional")
+            this.configureParameter("sensor.peltec", "second_pump|b_p2", "optional")
```

Comment:
is |b_p2 really needed? it shall work without

Reply:
Honestly, don't know. Copy-pasta from BioTec. Life has taught me to keep as-is until I know what it means :)

Reply:
ok, merged, I will clean that in next release if I will remember that :) thanks a lot for contribution

Reply:
Thanks for a solid base to work with!